src/util.js

```diff
@@ -6,18 +6,20 @@ export function submitForm(form, { headers, cors, strict } = {}) {
 	let method = form.getAttribute("method");
 	method = method ? method.toUpperCase() : "GET";
 	let uri = form.getAttribute("action");
-	let payload = serializeForm(form);
+	let body;
 
 	if(method === "GET") {
 		if(uri.indexOf("?") !== -1) {
 			throw new Error("query strings are invalid within `GET` forms' " +
 					"`action` URI; please use hidden fields instead");
 		}
-		uri = [uri, payload].join("?");
+		uri = [uri, serializeForm(form, new StringParams())].join("?");
+	} else if(form.getAttribute("enctype") === "multipart/form-data") {
```

Comment:
While generally speaking, this heuristic should be fine, there's also the option to [override the form's attribute via `formenctype`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#Attributes_for_form_submission) - not sure whether that's something we could detect or have to account for; seems like a very rare edge case, at least. (See README for [related caveats](https://github.com/FND/hijax-form#caveats).)